pkg/plugins/manager/store/store.go

```diff
@@ -27,6 +29,12 @@ func ProvideService(pluginRegistry registry.Service, pluginSources sources.Regis
 	return New(pluginRegistry), nil
 }
 
+func (s *Service) Run(ctx context.Context) error {
+	<-ctx.Done()
+	s.shutdown(ctx)
```

Comment:
Hmm should the store be responsible for shutting down plugins?

Reply:
It's responsible for loading them so for the current state I would say yes 😅

Reply:
Yes, but using the loader :) Maybe it should call loader.Unload then? :shrug: No big deal :)

Reply:
It's a good point you make. You see it's a bit confusing right now because `Unload` will actually remove from disk (it is only called from the uninstall flow), whereas `Load` works off what is already on disk / available - so they're actually mixing things a bit. This ties in with another idea I have so I'll think on this a bit more before merging!

Reply:
Don't want to block. Feel free to merge and continue in a new PR